src/components/accordion-group/accordion/bl-accordion.css

```diff
@@ -0,0 +1,81 @@
+:host {
+  display: block;
+}
+
+.accordion {
+  --border: 1px solid var(--bl-color-neutral-lighter);
+  --default-radius: var(--bl-size-2xs);
+  --radius-top-left: var(--bl-accordion-radius-top-left, var(--default-radius));
+  --radius-top-right: var(--bl-accordion-radius-top-right, var(--default-radius));
+  --radius-bottom-right: var(--bl-accordion-radius-bottom-right, var(--default-radius));
+  --radius-bottom-left: var(--bl-accordion-radius-bottom-left, var(--default-radius));
```

Comment:
I added it for customization, but if there is a problem, we can remove it.

Reply:
I think we can ask @buseselvi to should we allow customization for borders. If it is ok, we can use only one variable instead of 4 corner and we can pass values for every corner in one variable. Example case exists in our codebase

```
 //bl-alert.css
  border-radius: var(--bl-border-radius-l);
```

Reply:
I think it would be better to define one variant for the radius 🙏